ibexOverview.js

```diff
@@ -0,0 +1,140 @@
+var PORT = 60000;
+var HOST = "http://dataweb.isis.rl.ac.uk" 
+//$('.btn').button();
+
+function refresh() {
+	$.ajax({
+		url: HOST + ":" + PORT + "/",
+		dataType: 'jsonp',
+		data: {"Instrument": "all"},
+		timeout: 1000,
+		error: function(xhr, status, error){ 
+			window.alert("Error: JSON not read");
+		},
+		success: function(data){ 
+			//window.alert(JSON.stringify(data));
+			display_data(data);
+		}
+	});
+}
+
+function clearBox(elementID){
+    document.getElementById(elementID).innerHTML = "";
+};
+
+function sortObject(o) {
+    var sorted = {},
+    key, a = [];
+
+    for (key in o) {
+        if (o.hasOwnProperty(key)) {
+            a.push(key);
+        };
+    };
+
+    a.sort();
+
+    for (key = 0; key < a.length; key++) {
+        sorted[a[key]] = o[a[key]];
+    }
+    return sorted;
+}
+
+function close_window(){
+	clearBox("new_window");
+};
+
+function create_new_window(){
+	clearBox("new_window");
+	var newWindow = document.createElement("div");
+	var newWindowStyle = document.createAttribute("class");
+	newWindowStyle.value = "col-sm-12 well";
+	newWindow.setAttributeNode(newWindowStyle);
+	newWindow.id = "internal";
+	document.getElementById("new_window").appendChild(newWindow);
+	var closeButton = document.createElement("button");
+	closeButton.innerHTML = "x";
+	var blockListStyle = document.createAttribute("onclick");
+    blockListStyle.value = "close_window()";
+	closeButton.setAttributeNode(blockListStyle);
+	document.getElementById("internal").appendChild(closeButton);	
```

Comment:
Some commenting here would be beneficial so we know what is being created and why